src/pages/index.js

```diff
@@ -1,97 +0,0 @@
-import React, { Fragment, useMemo } from 'react';
-import { graphql } from 'gatsby';
-import { global } from '@storybook/design-system';
-
-import { IndexScreen } from '../components/screens/IndexScreen/IndexScreen';
-
-const { GlobalStyle } = global;
-
-const HomePage = ({ data }) => {
-  const {
-    storybookProjects,
-    dxData: {
-      npmDownloads,
-      latestPost,
-      twitterFollowerCount,
-      discordMemberCount,
-      githubContributorCount,
-      youTubeSubscriberCount,
-    },
-  } = data;
-
-  const projects = useMemo(
-    () =>
-      storybookProjects.projects.map(({ slug, logo, ...project }) => ({
-        logoUrl: logo.url,
-        height: logo.height,
-        width: logo.width,
-        projectUrl: `https://storybook.js.org/showcase/${slug}`,
-        ...project,
-      })),
-    [storybookProjects]
-  );
-
-  return (
-    <Fragment>
-      <GlobalStyle />
-      <IndexScreen
-        latestBlogPost={latestPost}
-        projects={projects}
-        npmDownloads={npmDownloads}
-        twitterFollowerCount={twitterFollowerCount}
-        discordMemberCount={discordMemberCount}
-        githubContributorCount={githubContributorCount}
-        youTubeSubscriberCount={youTubeSubscriberCount}
-      />
-    </Fragment>
```

Comment:
@winkerVSbecks it was being added here in the template file that was deleted.

It's used in all other screen components and we don't get all of our style rules without it :(